helm/values-uat.yaml

```
@@ -75,6 +75,9 @@ microservice-chart:
     OTEL_EXPORTER_OTLP_ENDPOINT: "http://otel-collector.elastic-system.svc:4317"
     OTEL_LOGS_EXPORTER: none
     OTEL_TRACES_SAMPLER: "always_on"
+    ONEIDENTITY_BASE_URL: "https://mock-u.example.com"                   # TODO: adjust for uat environment
+    ONEIDENTITY_REDIRECT_URI: "https://mock-u.example.com/client/login"  # TODO: adjust for uat environment
+    ONEIDENTITY_CLIENT_ID: "oneidentity-client-id"
```

```suggestion
    ONEIDENTITY_CLIENT_ID: "DlBY1GBNsl-w33g8Cpi1MXaMFNCxxvgg0KQPhEvmZ4I"
```
keep calm, it's not a secret :-)